definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module "lodash" {
-  declare type HigherOrderFunctionOptional = (funcs?: Array<Function>) => Function
+  declare type HigherOrderFunctionOptional = (...funcs?: Array<Function>) => Function
```

Comment:
this is a big change, is this correct?

Reply:
Oops, yes. Just checked the lodash docs. Before it was also possible with `...funcs`, but now it seems to only work with `funcs?: Array`. I originally wanted to make it more compatible with `$Compose` and `$ComposeReverse`.

Looking into it again.

Reply:
`compose` and `pipe` tests fail, if I remove the var args again. I'm not sure why. Probably in connection with your other comment in line 3113.

In the class before, compose was defined like this:
```javascript
  compose: $Compose;
  compose(funcs: Array<Function>): Function;
```
now I tried to recreate it in a module with `declare` like this:
```javascript
declare type HigherOrderFunction = (funcs: Array<Function>) => Function

declare export var compose: HigherOrderFunction | $Compose;
```
But that doesn't seem to be the equivalent of the above class-way

Reply:
001de7a that worked! thanks